docs/portal/registry-users/managing-registry-users.md

```diff
@@ -12,7 +12,7 @@ The following figure shows the key elements on the page, and the table following
 
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
-1                                                                                                       | To filter registry users, type characters in this field and then press the Enter key. All registry users that do not contain the typed characters are hidden. Filtering is not case-sensitive. To remove the filter, click the **Reset** button.                                             |
-| 2                        | Each registry user appears on its own row. Clicking a registry user allows you to view basic information about the registry user and the projects associated with the user.             |
-| 3                        | The **Actions** drop-down list on each row has options to [update the registry user](</docs/portal/registry-users/updating-users.md>), [change the registry user's password](</docs/portal/registry-users/changing-passwords.md>), and [delete the registry user](</docs/portal/registry-users/deleting-users.md>).                                                                          |
-| 4                        | The **+ Add New User** button allows you to [add registry users](</docs/portal/registry-users/adding-users.md>).          |
\ No newline at end of file
+<p style="text-align: center;">1</p>                                                                                                       | To filter registry users, type characters in this field and then press the Enter key. All registry users that do not contain the typed characters are hidden. Filtering is not case-sensitive. To remove the filter, click the **Reset** button.                                             |
```

Comment:
The preview of this list shows there are errors. Please check

![image](https://user-images.githubusercontent.com/43203744/105329371-0cdb6580-5c0c-11eb-910f-3e2be2916263.png)